torch/onnx/symbolic_opset11.py

```diff
@@ -117,3 +117,40 @@ def unique_dim(g, self, dim, sorted, return_inverse, return_counts):
 
 def round(g, self):
     return g.op("Round", self)
+
+
+def arange(g, *args):
```

Comment:
there might be a way to simplify the code, but if we are keeping the same structure as arange() in symbolic_opset9.py would it be possible to refactor the code to avoid duplicating the code?

Reply:
nvm I think it will make the code more complex rather then simplify it

Reply:
Yeah, I thought the same. There would be too many if/else cases.